build-docker-images.sh

```diff
@@ -1,41 +0,0 @@
-#!/bin/bash
```

Comment:
hmm any reason to remove this file? this could still help users if they wanted to build themselves right?

Reply:
Let me add a README to help with this.

Reply:
Added README, let me know what you think.

Reply:
nice!